app/views/booths/index.html.haml

```diff
@@ -0,0 +1,44 @@
+.container
+  .row
+    .col-md-12.page-header
+      %h1
+        Your booth requests for
+        = @conference.title
+      .pull-right
+        - if can? :create, Booth
+          = link_to 'Add Booth', new_conference_booth_path(@conference.short_title), class: 'button btn btn-primary'
+  .row
+    .col-md-12
+      .margin-booth-table
+        %table.table.table-striped.table-bordered.table-hover.datatable
```

Comment:
We don't expect the user to submit more than a couple of booth requests, so making this table a datatable might be unnecessary.